tests/AttributeTest.php

```diff
@@ -293,6 +293,22 @@ class Foo
                 'ignored_issues' => [],
                 'php_version' => '8.2',
             ],
+            'override' => [
+                'code' => '<?php
+
+                    namespace OverrideAttribute;
+
+                    use Override;
+
+                    class HelloWorld {
+                        #[Override]
+                        public function __invoke() {}
+                    }
```

Comment:
Fyi, this is code that should actually error, because there's no parent method to override.

Reply:
You are completely right. However, this particular piece of code would throw a PHP fatal error (i.e. `HelloWorld::__invoke() has #[\Override] attribute, but no matching parent method exists`) - not a psalm error per se. The PR, as is, creates the `#[Override]` core attribute so that psalm does not throw an `InvalidAttribute` error. That being said, I am happy to fix this if the maintainers agree on doing so - I definitely think it is a good idea.